### PR TITLE
Update Contrib to use latest version of Otel Collector.

### PR DESCRIPTION
**Pull request by ccaraman**
Pulls in the latest release(Alpha v0.2.6) from OpenTelemetry Collector.

Ran tests locally and they pass.

### CODE REVIEW

**Review by pjanotti — APPROVED** [review]
LGTM

**Review by tigrannajaryan — COMMENTED** [review]
Please update all other go.mod files for all in this repo components. We keep them all on the same core version.

File: exporter/carbonexporter/go.mod [hunk 1]
```diff
@@ -3,8 +3,9 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/carbon
 go 1.13
 
 require (
+	github.com/VividCortex/gohistogram v1.0.0 // indirect
 	github.com/census-instrumentation/opencensus-proto v0.2.1
-	github.com/open-telemetry/opentelemetry-collector v0.2.4-0.20200122010738-1a92a7ea5aea
+	github.com/open-telemetry/opentelemetry-collector v0.2.6
```
Comment (ccaraman) [inline exporter/carbonexporter/go.mod]: @pjanotti want to double check if this version of the collector is good for the carbon exporter?

Comment (ccaraman) [reply exporter/carbonexporter/go.mod]: Synced offline - this version is good!

**Comment by ccaraman** [comment]
@tigrannajaryan - updated all the go mods I could find. Ready for another pass.

**Review by tigrannajaryan — APPROVED** [review]
@ccaraman please also do `make gotidy` if not already done.

**Pull request merged**